tests/ui/extern-flag/no-force-extern.stderr

```diff
@@ -0,0 +1,9 @@
+error: `#[panic_handler]` function required, but not found
+
+error: language item required, but not found: `eh_personality`
```

Comment:
Specific output here may be target-specific and fail on CI, I suggest checking other tests that report the same errors and repeat what they do.

Reply:
I think I've addressed this. `dont-check-compiler-stderr` + `error-pattern` to hone in on just the panic handler case, since `eh_personality` isn't required for `-Cpanic=abort`. Let me know if this isn't what you meant!